### ISSUE TITLE
assertDataFrame has min.rows bug

### ISSUE THREAD

**Issue by berndbischl** [mllg/checkmate#47]
```

d = iris[,-(1:5)]
# d = data frame with 0 columns and 150 rows
nrow(d)
#  150
assertDataFrame(d, min.rows=1)
Error in assertDataFrame(d, min.rows = 1) : 
  Assertion on 'd' failed: Must have at least 1 rows, but has 0 rows
```

**Comment by berndbischl** [comment]
The error seems to be here:

```
/* FIXME: I am unsure but there seems to be a bug in R C API
 * ncols(x) does not work for data.frames
 * so we now have our own little ncol / nrow wrappers....... :(
*/
R_len_t get_nrows(SEXP x) {
  if (isFrame(x))
    return length(x) == 0 ? 0 : length(VECTOR_ELT(x, 0));
  else
    return nrows(x);
}
```


Just because the df has 0 cols, does not imply it has 0 rows?

**Comment by mllg** [comment]
Fixed in 3504a07cf53282f2802b68a7ff535b8a7b1046a5.
